libs/akita/src/lib/trackIdChanges.ts

```diff
@@ -0,0 +1,52 @@
+import { EntityState, EntityStore, getEntityType, getIDType, ID, QueryEntity } from '@datorama/akita';
```

Comment:
Expose it in public_api

Reply:
I added the export. There was also an import error by using '@datorama/akita' instead of relative paths.